features/HomeTab/AdvancedFilter/AdvancedFilter.tsx

```diff
@@ -1,7 +1,223 @@
-import { Text } from "react-native";
+import React, { useEffect, useState } from "react";
+import { ScrollView, StyleProp, View, ViewStyle } from "react-native";
 
-const AdvancedFilter = () => {
-  return <Text>AdvancedFilter</Text>;
+import Accomodation from "./Accomodation/Accomodation";
+import Amenities from "./Amenities/Amenities";
+import Footer from "./Footer/Footer";
+import Header from "./Header/Header";
+import PriceRange from "./PriceRange/PriceRange";
+import Rating from "./Rating/Rating";
+import RoomsAndBeds from "./RoomsAndBeds/RoomsAndBeds";
+import TypeOfPlace from "./TypeOfPlace/TypeOfPlace";
+
+interface BasicFilterProps {
+  route: any;
+  navigation: any;
+  bottomTabsRoute: any;
+  bottomTabsNavigation: any;
+}
+
+export type ButtonsType = {
+  value: string;
+  label: string;
+  style?: StyleProp<ViewStyle>;
+}[];
+
+const typeOfPlaceButtons: ButtonsType = [
+  {
+    value: "0",
+    label: "Any type",
+  },
+  {
+    value: "1",
+    label: "Room",
+  },
+  {
+    value: "2",
+    label: "Entire home",
+  },
+];
+
+const roomsAndBedsButtons: ButtonsType = [
+  { value: "0", label: "Any" },
+  { value: "1", label: "1" },
+  { value: "2", label: "2" },
+  { value: "3", label: "3" },
+  { value: "4", label: "4" },
+  { value: "5", label: "5" },
+  { value: "6", label: "6" },
```

Comment:
I'd suggest setting the label to "6+", otherwise it seems that apartments with eg. 7 beds is not reachable

Reply:
Good point, fixed